verifyEmail.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <?php require_once('universal.inc') ?>
+        <title>ODHS Medicine Tracker | Change Password</title>
```

Comment:
The title needs to change to say Stafford Junction instead of ODHS Medicine Tracker